src/lib/core/datetime.h

```diff
@@ -9,6 +9,7 @@
 #include <stdint.h>
```

Comment:
commit "datetime: validate unpacked data"

> NO_DOC=bugfix
NO_CHANGELOG=bugfix
NO_TEST=see the other commit for fuzzer

I'd rather add a changelog.